lib/backend/firestore/firestorebk.go

```diff
@@ -293,6 +310,18 @@ func New(ctx context.Context, params backend.Params) (*Backend, error) {
 	if !cfg.DisableExpiredDocumentPurge {
 		go RetryingAsyncFunctionRunner(b.clientContext, linearConfig, b.Logger, b.purgeExpiredDocuments, "purgeExpiredDocuments")
 	}
+
+	l.Debug("Waiting for event watcher to start")
+	select {
+	case <-ctx.Done():
+		l.Error("Timed out waiting for event monitor")
+		return nil, ctx.Err()
+
+	case <-b.watchStarted.Done():
```

Comment:
I think the `watchStarted` context is something that was recently deprecated, and @fspmarshall was planning to remove from all of the backends. We didn't want `New` to block, and instead wanted watchers to rely on the `OpInit` event to know when the watch is ready.

With this change, the firestore backend would be the only one using this context, even though it still exists in the code for the other backend types.

I think we ought to remain consistent and either use this approach for all backends or for no backends.

Reply:
OK, will revert

Reply:
Deleted entirely